### ISSUE TITLE
[Bug] Texture Variant Clears When Pressing File

### ISSUE THREAD

**Issue by krple** [ewanhowell5195/MinecraftTitleGenerator#152]
**TL;DR**
Select a texture variant, press on files, reverts back to base Pre-made texture.

**Description:**
When selecting a texture with a texture variant, such as "beach," and then changing the texture variant to "sand," if you press on the "File" option, the texture resets to the original "beach" texture. This behavior prevents you from using the trick to apply more than one overlay.

**Steps to Reproduce:**

- Select a texture with a texture variant (e.g., "beach").
- Change the texture variant to a different one (e.g., "sand").
- Press the "File" option.

**Expected Behavior:**
The selected texture should remain as the chosen variant ("sand").

**Actual Behavior:**
The texture resets to the original "beach" texture.

**Temporary Fix:**
There is a temporary workaround for this, it's to download the texture you want, and then upload the file in the "File" tab, but that makes your downloads folder get filled up.

**Comment by ewanhowell5195** [comment]
Fixed in version 1.4.0

It may be a while before this version actually releases
